common/client_config_v1.go

```diff
@@ -14,4 +14,5 @@ type ClientConfigV1 struct {
 	EdaClientCfg     clients.EigenDAClientConfig
 	MaxBlobSizeBytes uint64
 	PutRetries       uint
+	BackendsToEnable BackendsToEnable
```

Comment:
I personally think `BackendsToEnable` should live in the storageManager config instead of in any particular config. ALso think we might as well move `DisperseToV2` to storageManager config to make it live right next to BackendsToEnable (makes UX easier for user, since theyll be listed together in the flags help)